### PR TITLE
Refator JacksonUtils and JacksonUtilsTest

### LINKED ISSUE #364: Refator JacksonUtils and JacksonUtilsTest
**Why do you want to add this feature?**
1. remove the redundant logic of LOG in JacksonUtils.
2. refactor using assertThrows instead of try catch fail.

@SkyeBeFreeman

### PR DESCRIPTION
**Pull request by cheese8**
## PR Type

Refactoring 

## Describe what this PR does for and how you did.
Fix #364

### CODE REVIEW

File: spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/JacksonUtils.java [hunk 1]
```diff
@@ -79,9 +74,6 @@ public static Map<String, String> deserialize2Map(String jsonStr) {
 			return new HashMap<>();
 		}
 		catch (JsonProcessingException e) {
-			LOG.error(
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/JacksonUtils.java:82]: `RuntimeException` may be lost in a running thread if not be catched.

File: spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/JacksonUtils.java [hunk 2]
```diff
@@ -56,7 +52,6 @@ public static <T> String serialize2Json(T object) {
 			return OM.writeValueAsString(object);
 		}
 		catch (JsonProcessingException e) {
-			LOG.error("Object to Json failed. {}", object, e);
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/JacksonUtils.java:59]: `RuntimeException` may be lost in a running thread if not be catched.

**Comment by SkyeBeFreeman** [comment]
You need and info of this PR to CHANGELOG in the root of project. Will you cherry-pick to branch of 2020 and 2021?

**Comment by cheese8** [comment]
> You need add info of this PR to CHANGELOG in the root of project. Will you cherry-pick to branch of 2020 and 2021?

OK

**Pull request merged**